src/cartridges/int_adyen_SFRA/cartridge/scripts/hooks/payment/processor/middlewares/handle.js

```diff
@@ -25,12 +25,15 @@ function convertToSfccCardType(paymentInformation, paymentInstrument) {
     sfccCardType;
 
   if (paymentInformation.creditCardToken) {
+    const firstTwoDigitsFromCurrentYear = Math.floor(
```

Comment:
Can we create a helper function for this? So we can potentially reuse this in the future if needed